packages/flutter/lib/src/material/divider.dart

```diff
@@ -36,37 +36,52 @@ import 'theme.dart';
 /// ![](https://flutter.github.io/assets-for-api-docs/assets/material/divider.png)
 ///
 /// ```dart
-/// Widget build(BuildContext context) {
-///   return Center(
-///     child: Column(
-///       children: <Widget>[
-///         Expanded(
-///           child: Container(
-///             color: Colors.amber,
-///             child: const Center(
-///               child: Text('Above'),
+///   Widget build(BuildContext context) {
+///     return Center(
+///       child: Column(
+///         children: <Widget>[
+///           Expanded(
+///             child: Container(
+///               color: Colors.amber,
+///               child: const Center(
+///                 child: Text('Above'),
+///               ),
 ///             ),
 ///           ),
-///         ),
-///         const Divider(
-///           color: Colors.black,
-///           height: 20,
-///           thickness: 5,
-///           indent: 20,
-///           endIndent: 0,
-///         ),
-///         Expanded(
-///           child: Container(
-///             color: Colors.blue,
-///             child: const Center(
-///               child: Text('Below'),
+///           const Divider(
+///             color: Colors.black,
+///             height: 20,
+///             thickness: 5,
+///             indent: 20,
+///             endIndent: 0,
```

Comment:
can remove this line since it defaults to 0

Reply:
Done